File: C#/BubbleSort.cs
using System;
namespace BubbleSort {
   class MySort {
      static void Main(string[] args) {
         int[] arr = { 78, 55, 45, 98, 13 };
         int temp;
         for (int j = 0; j <= arr.Length - 2; j++) {
            for (int i = 0; i <= arr.Length - 2; i++) {
               if (arr[i] > arr[i + 1]) {
                  temp= arr[i + 1];
                  arr[i + 1] = arr[i];
                  arr[i] = temp;
               }
            }
         }
         Console.WriteLine("Sorted:");
         foreach (int p in arr)
            Console.Write(p + " ");
         Console.Read();
      }
   }
}

File: DSA/Notes and examples/bubblesort.cpp
//repeatedly swap adjacent elements if they are in wrong order
// Online C++ compiler to run C++ program online
#include <iostream>
using namespace std;

void bubbleSort()
{
  int i;
  int arr[]={10,2,3,4,5};
  
  //for no.of iterations
  int counter=1;
  int temp;
  while(counter<5)
  {
      for(i=0;i<5-counter;i++)
      {
          if(arr[i]>arr[i+1])
          {
              //swapping
              temp=arr[i];
              arr[i]=arr[i+1];
              arr[i+1]=temp;
          }
      }
      //to jump to next iteration
      counter++;
  }
    cout<<"sorted array:\n";
    for(i=0;i<5;i++)
    cout<<arr[i]<<" \n";
  
}


int main() {
    int arr[]={10,2,3,4,5};
    // int i,j,temp;
    cout<<"given array is:\n";
    for(int i=0;i<5;i++)
    cout<<arr[i]<<" \n";
    bubbleSort();
    return 0;
}
